regulations/templates/regulations/tree/preamble_intro.html

```diff
@@ -65,14 +65,16 @@ <h4 class="small-header">{{ meta.primary_agency }}</h4>
     </p>
   </div>
 </li>
-{% for c in node.children %}
-  {% with node=c %}
-    {% include node.template_name %}
-  {% endwith %}
-{% endfor %}
-{% if meta.supporting_documents %}
+<div class="intro-content">
```

Comment:
I wonder if there's something we could do to the outer `ol`, maybe give it a class like "node-type-{{node_type}}`, which may solve this.